package.json

```
@@ -127,10 +137,13 @@
       "tests/**/*.{js,ts,tsx}"
     ]
   },
-  "dependencies": {},
+  "dependencies": {
+    "@babel/runtime": "^7.13.10"
+  },
```

```suggestion
  "dependencies": {},
```

I spent quite some effort to remove dependencies, so let's remove @babel/(plugin-transform-)runtime.